Library/Formula/vobcopy.rb

```diff
@@ -14,6 +11,11 @@ def install
                              "--with-lfs",
                              "--with-dvdread-libs=#{HOMEBREW_PREFIX}"
     system "make"
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/vobcopy -h 2>&1", 1)
```

Comment:
Well, It's been 4 years since it was last touched in Homebrew. If you're having runtime issues, I'm happy enough to see it boneyarded - Nothing in the core depends on it. CC @mikemcquaid